lib/form/js/repeatable.js

```diff
@@ -0,0 +1,266 @@
+/* eslint-env browser */
+/* global allPlugins, allInputs, allSettings, allIDs, allRepeatables, validate */
+
+(function formRepeatable() {
+  'use strict';
+
+  // Splits the string and pops the last element
+  var splitPop = function pop(input, splitter) {
+    var blocks = input.split(splitter);
+    blocks.pop();
+    blocks = blocks.join(splitter);
+
+    return blocks;
+  };
+
+  // Iterates through every instance of input-plugins
+  Object.keys(allSettings).forEach(function repeatableHandler(input) {
+    var idAdd = input + '--add',
+        idDelete = input + '--delete',
+        deleteHandler,
+        addHandler,
+        addButton;
+
+    deleteHandler = function deleteEventHandler(e) {
```

Comment:
You appear to be creating this function for each and every setting. Is that true? We really don't want to be creating functions inside of loops; can you have these functions built separately outside of the loop and just loop over the settings to apply the functions as needed?